src/modules/hip/hip_color_model_conversions.cpp

```diff
@@ -396,35 +262,8 @@ look_up_table_hip_batch(Rpp8u* srcPtr, Rpp8u* dstPtr, Rpp8u* lutPtr,rpp::Handle&
     Rpp32u max_height, max_width;
     max_size(handle.GetInitHandle()->mem.mgpu.csrcSize.height, handle.GetInitHandle()->mem.mgpu.csrcSize.width, handle.GetBatchSize(), &max_height, &max_width);
 
-#if defined (HIPRTC)
-
-    std::vector<size_t> vld{32, 32, 1};
-    std::vector<size_t> vgd{max_width, max_height, handle.GetBatchSize()};
-
-    handle.AddKernel("", "", "look_up_table.cpp", "look_up_table_batch", vld, vgd, "")(srcPtr,
-                                                                                       dstPtr,
-                                                                                       hipLutPtr,
-                                                                                       handle.GetInitHandle()->mem.mgpu.roiPoints.x,
-                                                                                       handle.GetInitHandle()->mem.mgpu.roiPoints.roiWidth,
-                                                                                       handle.GetInitHandle()->mem.mgpu.roiPoints.y,
-                                                                                       handle.GetInitHandle()->mem.mgpu.roiPoints.roiHeight,
-                                                                                       handle.GetInitHandle()->mem.mgpu.srcSize.height,
-                                                                                       handle.GetInitHandle()->mem.mgpu.srcSize.width,
-                                                                                       handle.GetInitHandle()->mem.mgpu.maxSrcSize.width,
-                                                                                       handle.GetInitHandle()->mem.mgpu.srcBatchIndex,
-                                                                                       channel,
-                                                                                       handle.GetInitHandle()->mem.mgpu.inc,
-                                                                                       plnpkdind);
-
-    hipFree(&hipLutPtr);
-
-
-#elif defined(STATIC)
-
     hip_exec_look_up_table_batch(srcPtr, dstPtr, hipLutPtr, handle, chnFormat, channel, plnpkdind, max_height, max_width);
 
```

Comment:
Add a hipFree(&hipLutPtr); after the hip_exec call since it was present in the HIPRTC version